src/util/routeConstants.ts

```diff
@@ -1,7 +1,7 @@
 export const HOME = '/';
-export const ABOUT = '/about';
+export const ABOUT = '#about-us';
 export const MEMBERS = '/members';
-export const EVENTS = '/events';
+export const EVENTS = '#events';
 export const NEWS = '/news';
 export const FORUM = '/forum';
```

Comment:
Let us remove the `MEMBERS`, `NEWS` and `FORUM` variables that are currently unused.

Reply:
Fixed via [fc880c9](https://github.com/reactdeveloperske/reactdevske-website/pull/105/commits/fc880c969889f1f06a8e2917347d5584f1391ff5)